src/Pagination/Pagination.test.jsx

```diff
@@ -0,0 +1,257 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import checkPropTypes from 'check-prop-types';
+
+import '../utils/reactResponsive.mock';
+import { breakpoints } from '../Responsive';
+
+import Pagination from './index';
+
+const baseProps = {
+  paginationLabel: 'pagination navigation',
+  pageCount: 5,
+  onPageSelect: () => {},
+};
+
+describe('<Pagination />', () => {
+  it('renders', () => {
+    const props = {
+      ...baseProps,
+    };
+    const wrapper = mount(<Pagination {...props} />);
```

Comment:
If possible, I'd recommend using [shallow](http://airbnb.io/enzyme/docs/api/shallow.html) instead of mount for tests, especially where you don't actually need any children to render for the test to run successfully. The places where you need, say, for a Button to render, you'd find the Button and call `.dive()` on it. My understanding is that it's best practice to use shallow over mount whenever possible. If you're interested in changing it and it's taking longer than you'd like, I'd be happy to let me know, since I've had to do it a few times so I should be able to help.

Reply:
I'll try using `shallow` here instead!